### ISSUE TITLE
SC1136 is issued for multiline conditions in `if` clauses with backslash linebreaking

### ISSUE THREAD

**Issue by AlexanderRevo** [koalaman/shellcheck#2101]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC1136
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash

test="abc"

if [[ "${test}" == "1" ]]\
    || [[ "${test}" == 2 ]]
then
    echo "ok"
fi
```

#### Here's what shellcheck currently says:
```
Line 5	SC1136: Unexpected characters after terminating ]]. Missing semicolon/linefeed?
```
#### Here's what I wanted or expected to see:

No warnings.

**Comment by koalaman** [comment]
This has now been fixed generally as part of better trailing argument handling in 8e332ce87. Thanks!
